src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -67,7 +67,7 @@ func main() {
         // {@TODO()} Handle error.
       }
     @else
-      err := {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
+      err = {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
```

Comment:
It's actually what we had before (and have published):
https://cloud-dot-devsite.googleplex.com/storage/docs/json_api/v1/bucketAccessControls/delete#examples

Not sure why that changed.